cmd/crypto/help.go

```diff
@@ -20,15 +20,15 @@ import "github.com/minio/minio/cmd/config"
 
 // Help template for KMS vault
 var (
-	Help = config.HelpKVS{
+	HelpVault = config.HelpKVS{
 		config.HelpKV{
 			Key:         KMSVaultEndpoint,
 			Description: `API endpoint e.g. "http://vault-endpoint-ip:8200"`,
 			Type:        "url",
 		},
 		config.HelpKV{
 			Key:         KMSVaultKeyName,
-			Description: `unique transit key name e.g. "my-minio-key"`,
```

Comment:
@aead this text needs to be preserved as reviewed by @abperiasamy

Reply:
Oh, so AB reviewed that. Because, honestly, those help descriptions are mostly not really helpful.
If I don't know what I should set here, how does `unique transit key name` help me?! I would need to know what `transit`(the Vault transit engine) is and it does not tell what this key (name) is used for. (Despite the fact that it's not really correct that this key name has to be unique, a.s.o....

Reply:
This is part of vault subsystem so naturally the help is vault specific we don't need to elaborate that this is again vault key name.